docs/supported-tools.rst

```diff
@@ -0,0 +1,260 @@
+
+When an incompatible change happens in one of these tools or dependencies,
+we need to change our code as well to keep our features working.
+This is done with backwards compatibility in mind,
+but sometimes is hard to keep compatibility with old and new versions at the same time.
+In order to continue moving forward on future development and features,
+we need to drop support for some versions.
+
+.. note::
+
+   Your existent documentation will always be kept online and working,
+   but **when support for a tool or dependency ends, new builds may fail**.
+
+.. contents:: Contents
+   :local:
+   :depth: 3
+
+End of support policy
+---------------------
+
+Our policy defines how long a given tool or dependency is considered supported.
+Read the Docs will contact all users when an end of support date is close,
+after that date your builds may start failing and you will need to upgrade in order to receive support.
+For :doc:`/commercial/index` we provide an extended support of six months after the official end of support date.
+
+This is how we choose an end of support date:
+
+- For tools that define an :abbr:`EOL (End Of Life)` date, we try to follow that date for our policy.
+- For tools that release their versions incrementally without an EOL date,
+  we choose a date based on:
+
+  - A version with breaking changes is released (usually a mayor version update).
+  - The date since it was last updated.
+  - Its usage on our platform.
+
+.. note::
+
+   Some recent versions may be supported, but aren't listed on these tables
+   (specially minor updates from supported versions).
+   Contact us if you have doubts.
+
+Default dependencies
+--------------------
+
+We install some dependencies in order to build your project without specifying their dependencies.
+In the past we used to set these dependencies to a specific version and update them after some time,
+but by doing so would break some builds and new projects wouldn't be able to use a new version by default.
+For this reason we are now installing their latest version (or latest supported version) by default.
+
+.. note::
+
+   In order to keep your builds reproducible,
+   it's highly recommended declaring its dependencies and versions explicitly.
+   See :doc:`/guides/reproducible-builds`.
+
+External dependencies (Python)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are the dependencies that are installed by default when using a Python environment:
+
+Sphinx:
+  Projects created before Oct 20, 2020 use ``1.8.x``.
+  New projects use the latest version.
+
+Mkdocs:
+  Projects created before April 3, 2019 (April 23, 2019 for :doc:`/commercial/index`) use ``0.17.3``.
+  Projects created before March 9, 2021 use ``1.0.4``.
+  New projects use the latest version.
+
+sphinx-rtd-theme:
+  Projects created before October 20, 2020 (January 21, 2021 for :doc:`/commercial/index`) use ``0.4.3``.
+  New projects use the latest version.
```

Comment:
I'm not sure about not installing the theme in the future as well. Like, it's an external dependency, but it's a "core" product as well.

Reply:
Not installing it means fixing the “what happened?” issue when you find out that the rendering of your documentation locally does not match what Read The Docs show.

However, it’s a great theme, and if it’s not installed by default, it will be somewhat harder for people to discover it, or at least it will become less popular due to people tending to leave whatever is default.

Reply:
@Gallaecio we are trying to do less "magic" for users in favor of being more explicit and having reproducible builds locally. In this case users would need to add the theme to their requirements, we would update or add guides to help users with this.